spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -2270,8 +2270,8 @@ def __recover_from_error(self, exception):
         if not unsuccessful_cores:
             for executable_type, core_subsets in \
                     self._data_writer.get_executable_types().items():
-                failed_cores = transceiver.get_cpu_infos(
-                    core_subsets, executable_type.end_state, False)
+                failed_cores = transceiver.get_cores_in_states(
```

Comment:
Should this be "not in state" since it was include=False?

Reply:
agreed